rpm/harbour-nayttamo.spec

```diff
@@ -65,6 +65,7 @@ desktop-file-install --delete-original       \
 %defattr(-,root,root,-)
 %{_bindir}
 %{_datadir}/%{name}
+%{_datadir}/%{name}/translations
```

Comment:
did Qt Creator modify these or was there a bug? I'm a bit confused with these two files still... I've only used them to increase version numbers so far :sweat_smile:

Reply:
This change is actually not needed. I had some issues with translations until I figured out that running the app from Sailfish SDK deploy makes translations fail and this remained from my testting. I can remove this is needed.